tests/stub/routing/scripts/v4x3/router_yielding_empty_response_then_shuts_down.script

```diff
@@ -5,7 +5,4 @@ C: HELLO {"scheme": "basic", "credentials": "c", "principal": "p", "user_agent":
 S: SUCCESS {"server": "#SERVER_AGENT#", "connection_id": "bolt-123456789"}
 C: ROUTE #ROUTINGCTX# [] "adb"
 S: SUCCESS { "rt": { "ttl": 1000, "servers": []}}
-{?
-    C: GOODBYE
-    S: <EXIT>
-?}
+   <EXIT>
```

Comment:
It seems a bit awkward.

Reply:
I don't know it this was intentional, but the test used to have `!: AUTO GOODBYE` before I changed it to this optional GOODBYE message with #193. That change was not equivalent as adding the optional block made the server wait after sending the last `SUCCESS` message to see if a GOODBYE would follow. This actually broke the tests.

Reply:
So, the `!: AUTO GOODBYE` was doing a bit a more than expected.

Reply:
Kind of. But still, relying on the server to shut down for a test to pass is probably not a good thin in most cases (unless you're explicitly simulating a failing server). So I think that having explicit auto lines and an explicit `GOODBYE` is a good thing and helps to avoid oversights.